scripts/release-notes.py

```diff
@@ -165,6 +165,7 @@ def lookup_person(name, email):
     'performance improvement': "Performance improvements",
     'bug fix': "Bug fixes",
     'security update': "Security updates",
+    'cluster virtualization': "Cluster virtualization and multi-tenancy",
 }
 
 # Order in which to show the sections.
```

Comment:
@knz In which order should we specify `cluster virtualization` in the list?

Reply:
I don't know yet. Your team still needs to discuss this.

Reply:
@lnhsingh Where do you want this new item to live? This is our current order:

```
    'backward-incompatible change',
    'security update',
    'general change',
    'enterprise change',
    'sql change',
    'ops change',
    'cli change',
    'ui change',
    'bug fix',
    'performance improvement',
    'build change',
```

Reply:
Maybe after `ops change`?

Reply:
@knz For congruency sake, can you add this in line 179?
```
    'cluster virtualization',
```

Reply:
Done.